holoviews/operation/datashader.py

```diff
@@ -297,8 +297,11 @@ def _get_agg_params(self, element, x, y, agg_fn, bounds):
         params = dict(get_param_values(element), kdims=[x, y],
                       datatype=['xarray'], bounds=bounds)
 
-        kdim_list = '_'.join(str(kd) for kd in params['kdims'])
-        vdim_prefix = self.vdim_prefix.format(kdims=kdim_list)
+        if self.vdim_prefix is not None:
```

Comment:
Not real different but no point doing the kdims join if there is no prefix string:

```suggestion
        if self.vdim_prefix:
```

Reply:
Sure!